_drafts/2016-03-22-software-review.Rmd

```diff
@@ -0,0 +1,218 @@
+---
+name: software-review
+layout: post
+title: "Onboarding at ROpenSci: A Year in Reviews"
+date: 2016-03-22
+authors:
+  - name: Noam Ross
+    url: http://twitter.com/noamross
+  - name: Carl Boettiger
+  - name: Jenny Bryan
+  - name: Scott Chamberlain
+  - name: Rich FitzJohn
+  - name: Karthik Ram
+tags:
+  - software
+  - review
+  - infrastructure
+---
+
+Code review, in which peers manually inspect the source code of software
+written by others, is widely recognized as one of the best tools for finding
+bugs in software. Code review is relatively uncommon in *scientific* software
+development, though. Scientists, despite being familiar with the process of
+peer review, often have little exposure code review due to lack of training and
+historically little incentive to share the source code from their research. So
+scientific code, from one-off scripts to reusable R packages, is rarely subject
+to review. Most R packages are subject only to the automated checks required by
+CRAN, which primarily ensure that packages can be installed on multiple systems.
+As such, The burden is on software users to discern well-written and efficient
+packages from poorly written ones.
+
+ROpenSci is a community of developer-scientists, creating R packages for other
+scientists, and our package contributors have a mix of backgrounds. We aim to
+serve our users with high-quality software, but also promote best practices
```

Comment:
the `but` here seems to put high quality software and promoting best practices at odds, or maybe i'm not reading it right